src/utils/DisplayUtils.java

```diff
@@ -0,0 +1,51 @@
+package utils;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import model.User;
+import model.UserRegistry;
+
+public class DisplayUtils {
+
```

Comment:
Why not name like Dashboard, it will be used to showing dashboard section.

Reply:
I thought dashboard would be more of a UI service that could internally call a display formatter and that this displayUtil could be used by multiple calls from other services as well.

Reply:
this is not exactly util so why name it as util?